perllib/FixMyStreet/DB/Result/Comment.pm

```diff
@@ -264,6 +263,7 @@ sub meta_line {
     }
 
     if (!$update_state and $self->problem_state) {
+      my $state;
```

Comment:
This is fine, just to note you could define and set it by combining with the line below.